rules/instance_network_interface_rules.yaml

```diff
@@ -0,0 +1,13 @@
+rules:
```

Comment:
@ahoying do you think we could do folder-level rules for firewall rules might be something we can do in a followup PR?